server/models.py

```diff
@@ -98,3 +100,21 @@ def __init__(self, challenge, interest, time, lecture):
 
     def __repr__(self):
         return "<Engagement {}>".format(self.id)
+
+
+class CommentRating(db.Model):
+    user = db.Column(db.String(), primary_key=True)
+    lecture_id = db.Column(db.Integer, db.ForeignKey('lecture.id'), primary_key=True)
+    comment_id = db.Column(db.Integer, db.ForeignKey('comment.id'), primary_key=True)
+    rating = db.Column(db.Integer, nullable=False)
+
+    def __init__(self, rating, user, comment, lecture):
+        self.rating = rating
+        self.user = user
+        self.comment = comment
+        self.comment_id = comment.id
```

Comment:
I though so too, but I encountered some issues with it. I tried not setting _id now and it works, so I guess I did something wrong earlier. I'll push a fix.